lib/membrane_mp4/container/header.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Membrane.MP4.Container.Header do
+  @moduledoc """
+  A structure describing the header of the box.
+
+  The `content_size` field is equal to the box size minus the size of the header (8 bytes).
+  """
+
+  @enforce_keys [:name, :content_size]
+
+  defstruct @enforce_keys
+
+  @type t :: %__MODULE__{
+          name: atom(),
+          content_size: non_neg_integer()
+        }
+
+  @name_size 4
+  @size_size 4
+  @header_size @name_size + @size_size
+
+  @doc """
+  Parses the box's header.
```

Comment:
```suggestion
  Parses the header of a box.
```
It's hard to read, even in silence 😆

Reply:
xd